tests/test_cases/test_package/test_package.py

```diff
@@ -0,0 +1,23 @@
+"""
+A set of tests that demonstrate package access
+"""
+
+import logging
+import os
+
+import cocotb
+from cocotb import simulator
+from cocotb.handle import SimHandle
+
+
+@cocotb.test()
+async def test_params(dut):
+    """Test package parameter access"""
+    tlog = logging.getLogger("cocotb.test")
+
+    tlog.info("Checking Parameters:")
+    assert dut.seven_int.value == 7
+    pkg_handle = simulator.get_package_handle("cocotb_package_pkg")
+    pkg = SimHandle(pkg_handle)
+    pkg._discover_all()
```

Comment:
I don't understand why `_discover_all()` is necessary here.  Without it I get:
```
AttributeError: cocotb_package_pkg:: contains no object named five_int
```
Not a big deal since I can just discover each package at init time.  But if this indicates I'm doing something dumb, please enlighten me.